examples/edit_text.rs

```diff
@@ -0,0 +1,509 @@
+use glazier::kurbo::Size;
+use glazier::{
+    keyboard_types::Key,
+    text::{
+        Action, Affinity, Direction, Event, HitTestPoint, InputHandler, Movement, Selection,
+        VerticalMovement,
+    },
+    Application, IdleToken, KeyEvent, Region, Scalable, TextFieldToken, WinHandler, WindowHandle,
+};
+use parley::{FontContext, Layout};
+use std::any::Any;
+use std::borrow::Cow;
+use std::cell::RefCell;
+use std::ops::Range;
+use std::rc::Rc;
+use unicode_segmentation::GraphemeCursor;
+use vello::util::{RenderContext, RenderSurface};
+use vello::Renderer;
+use vello::{
+    glyph::{
+        pinot::{types::Tag, FontRef},
+        GlyphContext,
+    },
+    kurbo::{Affine, Point, Rect},
+    peniko::{Brush, Color, Fill},
+    Scene, SceneBuilder,
+};
+
+const WIDTH: usize = 2048;
+const HEIGHT: usize = 1536;
+const FONT_SIZE: f32 = 36.0;
+const TEXT_X: f64 = 100.0;
+const TEXT_Y: f64 = 100.0;
+
+fn main() {
+    let app = Application::new().unwrap();
+    let mut window_builder = glazier::WindowBuilder::new(app.clone());
+    window_builder.resizable(true);
+    window_builder.set_size((WIDTH as f64 / 2., HEIGHT as f64 / 2.).into());
```

Comment:
Why divide by 2?

Reply:
This is another copy from `shello.rs` :/